.reposync.yml

```diff
@@ -0,0 +1 @@
+
```

Comment:
If turning on reposync, should also sync the files as described in https://github.com/Particular/NServiceBus.Storage.MongoDB/pull/222#issuecomment-924977081 to avoid another PR needing to pass CI later.